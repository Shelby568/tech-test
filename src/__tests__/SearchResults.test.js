import React from 'react';
import { render } from '@testing-library/react';
import SearchResult from '../components/SearchResults';

describe('SearchResults', () => {
    const { asFragment } = render(<SearchResult />)

    it('makes sure component renders correctly', () => {
        expect(asFragment()).toMatchSnapshot();
    });
    
});