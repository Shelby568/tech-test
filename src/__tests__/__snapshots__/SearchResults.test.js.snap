// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchResults makes sure component renders correctly 1`] = `
<DocumentFragment>
  <div
    class="images"
  >
    <img
      alt="spaceimage"
      class="result-image"
      src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
    />
    <img
      alt="spaceimage"
      class="result-image"
      src="https://images.unsplash.com/photo-1522030299830-16b8d3d049fe?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80"
    />
  </div>
</DocumentFragment>
`;
